libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -18,10 +19,13 @@ class NavigationOptions private constructor(
 
     fun distanceFormatter() = distanceFormatter
 
+    fun onboardRouterConfig() = onboardRouterConfig
+
     data class Builder(
         private var timeFormatType: Int = NONE_SPECIFIED,
         private var roundingIncrement: Int = ROUNDING_INCREMENT_FIFTY,
-        private var distanceFormatter: DistanceFormatter?
+        private var distanceFormatter: DistanceFormatter?,
+        private var onboardRouterConfig: MapboxOnboardRouterConfig
```

Comment:
We should make this nullable and add a default implementation.

Reply:
done